lib/sqs.js

```diff
@@ -15,11 +23,35 @@ const clientConfig = require('./client-config');
  */
 module.exports = ({ account, queuePrefix = '', queueSuffix = '', defaultVisibilityTimeout = 300 }) => {
   const SQS = new AWS.SQS(clientConfig);
+  const S3  = new AWS.S3(clientConfig);
+  
   Promise.promisifyAll(SQS);
+  Promise.promisifyAll(S3);
 
   function getQueueURL(name) {
     return `https://sqs.${AWS.config.region}.amazonaws.com/${account}/${queuePrefix}${name}${queueSuffix}`;
   }
+  
+  function getHoursFromNow(hours) {
+    const now = new Date();
+    return now.setHours(hours);
+  }
+  
+  function send({ queueName, payload, attrs }) {
```

Comment:
is this the old `send`?  unchanged?

Reply:
yeah thats unchanged, i needed to pull it out to reference it from within the `extendedSend`